### DISCUSSION TITLE
Can customized Element sidebar and columns be stored in Project Config?

### DISCUSSION THREAD

**Discussion by BenParizek** [craftcms/cms#9780] [Ideas] [answered]
When we create a new Section its details get saved in Project Config and we can push that between environments.

However, if you 'Customize' the Entries sidebar and place that new section in a different place or group a handful of Sections under a new sidebar header, those changes do not get saved in Project Config and have to be re-implemented on the live site.

Additionally, if you customize the columns that display for a Section in the Element listing these changes also do not get stored in the project config so must be re-implemented on the live site.

Is there a reason the sidebar and customization features behave this way? Or could this fall in the category of Feature Request?

I would find it nice to be able to customize the user experience of new Sections in the sidebar and listing columns for deployment.

**Comment by brandonkelly** [comment, marked as answer]
This was previously requested by #8616. We’re planning to make that change in Craft 4.
